snmp.yml

```diff
@@ -1565,3 +1565,946 @@ fortinet_fortigate:
+          type: PhysAddress48
+    - name: mtxrWlRtabTxPackets
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.6
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabRxPackets
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.7
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabTxRate
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.8
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabRxRate
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.9
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+
+    - name: mtxrWlApIndex
+      oid: 1.3.6.1.4.1.14988.1.1.1.3.1.1
+    - name: mtxrWlApOverallTxCCQ
+      oid: 1.3.6.1.4.1.14988.1.1.1.3.1.10
+      indexes:
+        - labelname: mtxrWlApIndex
+          type: Integer32
+      lookups:
+        - labels: [mtxrWlApIndex]
+          labelname: mtxrWlApSsid
+          oid: 1.3.6.1.4.1.14988.1.1.1.3.1.4
+    - name: mtxrWlApAuthClientCount
+      oid: 1.3.6.1.4.1.14988.1.1.1.3.1.11
+      indexes:
+        - labelname: mtxrWlApIndex
+          type: Integer32
+      lookups:
+        - labels: [mtxrWlApIndex]
+          labelname: mtxrWlApSsid
+          oid: 1.3.6.1.4.1.14988.1.1.1.3.1.4
+    - name: mtxrWlApTxRate
+      oid: 1.3.6.1.4.1.14988.1.1.1.3.1.2
+      indexes:
+        - labelname: mtxrWlApIndex
+          type: Integer32
+      lookups:
+        - labels: [mtxrWlApIndex]
+          labelname: mtxrWlApSsid
+          oid: 1.3.6.1.4.1.14988.1.1.1.3.1.4
+    - name: mtxrWlApRxRate
+      oid: 1.3.6.1.4.1.14988.1.1.1.3.1.3
+      indexes:
+        - labelname: mtxrWlApIndex
+          type: Integer32
+      lookups:
+        - labels: [mtxrWlApIndex]
+          labelname: mtxrWlApSsid
+          oid: 1.3.6.1.4.1.14988.1.1.1.3.1.4
+    - name: mtxrWlApClientCount
+      oid: 1.3.6.1.4.1.14988.1.1.1.3.1.6
+      indexes:
+        - labelname: mtxrWlApIndex
+          type: Integer32
+      lookups:
+        - labels: [mtxrWlApIndex]
+          labelname: mtxrWlApSsid
+          oid: 1.3.6.1.4.1.14988.1.1.1.3.1.4
+    - name: mtxrWlApFreq
+      oid: 1.3.6.1.4.1.14988.1.1.1.3.1.7
+      indexes:
+        - labelname: mtxrWlApIndex
+          type: Integer32
+      lookups:
+        - labels: [mtxrWlApIndex]
+          labelname: mtxrWlApSsid
+          oid: 1.3.6.1.4.1.14988.1.1.1.3.1.4
+        - labels: [mtxrWlApBand]
+          labelname: mtxrWlApBand
+          oid: 1.3.6.1.4.1.14988.1.1.1.3.1.8
+    - name: mtxrWlApNoiseFloor
+      oid: 1.3.6.1.4.1.14988.1.1.1.3.1.9
+      indexes:
+        - labelname: mtxrWlApIndex
+          type: Integer32
+      lookups:
+        - labels: [mtxrWlApIndex]
+          labelname: mtxrWlApSsid
+          oid: 1.3.6.1.4.1.14988.1.1.1.3.1.4
+    - name: mtxrHlCoreVoltage
+      oid: 1.3.6.1.4.1.14988.1.1.3.1
+    - name: mtxrHlTemperature
+      oid: 1.3.6.1.4.1.14988.1.1.3.10
+      indexes:
```

Comment:
Can you elaborate on why this means it should be removed? It is being reported appropriately in my testing.